decide/booth/templates/booth/booth.html

```diff
@@ -263,26 +232,30 @@ <h2>[[ voting.question.desc ]]</h2>
                 },
                 decideSend(evt) {
                     evt.preventDefault();
-
-                    // Check if the selected option is 'Voto en Blanco'
-                    // Check if the selected option is 'Voto en Blanco'
-                    // const selectedOption = this.voting.question.options.find(opt => opt.number === this.selected);
-            
-
-                    // if (selectedOption.option === 'Voto En blanco') {
-                    //     this.showAlert("danger", "No puedes votar en blanco");
-                    //     return;
-                    // }
-
-                    var v = this.decideEncrypt(this.selected.toString());
-                                       
-                    var data = {
-                        vote: {a: v.alpha.toString(), b: v.beta.toString()},
-                        voting: this.voting.id,
-                        voter: this.user.id,
-                        token: this.token
+                    if (this.questionType === 'S'){
+                        var v = this.decideEncrypt(this.selected.toString());
+                        var data = {
+                            vote: {a: v.alpha.toString(), b: v.beta.toString()},
+                            voting: this.voting.id,
+                            voter: this.user.id,
+                            token: this.token
+                        }
```

Comment:
Could you also take in account here boolean votings? It is not on the stable branch yet but the feature is ready and will need to be taken into account as well. You can create an issue for this and assign it to the pr as a feature for boolean votings

Reply:
If it is not a problem, I think I will implemented in another branch with the priority voting also.